### ISSUE TITLE
[chm-v1-run-control] 100M Stage-C execution implementation — one-shot scientific screen, trigger withheld

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#990]
## Classification

`CHM_V1_100M_STAGE_C_EXECUTION_IMPLEMENTATION_TRIGGER_WITHHELD`

This issue is the separately required executable Stage-C run-control successor to #977, terminal Stage-B systems PASS #983, merged evaluator #986/#987, and completed run-control preparation #988/#989.

It authorizes implementation, CPU/CI validation, PR merge, and final launcher provenance binding for exactly one future Stage-C scientific attempt. It does **not** authorize creation of the trigger issue or GPU allocation until a final owner authority comment is posted after exact-main CI and a fresh seed/result-namespace audit.

## Starting authority

Re-audited immediately before creation:
- live `main`: `0e21e48ce7b86ca6b385bcafc6eb556f617d3fbb`
- live tree: `8d09e0723160ce0eaae7166bad1c2ba414e52abc`
- model contract blob: `tam_research/chm_v1_100m_scale.py` = `b9b141c0e52d4fd0fff28b12a3588b2adc659b8f`
- aligned evaluator blob: `tam_research/chm_v1_100m_stage_c_eval.py` = `863bd038e60da5511503adb0c8e1046a680ed3bd`
- run-control preparation blob: `tam_research/chm_v1_100m_stage_c_run_control_prep.py` = `bc4ed60885aaf991a2d6b9fe8f634ff973f3f722`
- #988 exact-main CI: PASS, run `35075003832`, job `104725212481`
- reserved scientific seed `977001` remains untouched/unconsumed
- reserved trigger title `[modal-chm-v1-100m-stage-c-988-seed-977001-v1]` has no existing issue at creation time

## Frozen scientific execution

Exactly one future attempt may eventually be authorized:
- scientific seed: `977001`
- LOCAL then EIEM-FLAT
- `33,554,432` training tokens/model (`67,108,864` pair total)
- `2,048` optimizer steps/model
- session length `1024`; micro-batch `4`; grad accumulation `4`; 16,384 tokens/step
- AdamW betas `(0.9,0.95)`, weight decay `0.1`, peak LR `3e-4`, grad clip `1.0`
- inherited cosine LR schedule; warmup `40` steps
- BF16 CUDA autocast; torch.compile disabled
- byte-identical train sample stream/order for LOCAL and EIEM via frozen generator seed `987001`
- audit checkpoints at steps `512/1024/1536/2048`; only final step 2048 is scientifically evaluated
- no resume, rescue, early stopping, checkpoint selection, or retry after attempt consumption

## Frozen evaluator

Use only `chm-v1-100m-heldout-aligned-v4` from #986/#987:
- probe seed `977301`
- validation seed `977302`
- bootstrap seed `977303`
- 128 cases/family; 512 total; 384 long-range
- 1,048,576 ordinary-language validation tokens/model
- 10,000 stratified paired-bootstrap resamples
- exact PASS/STOP gate from #986; no threshold changes

## Frozen resources / cost

Future execution target:
- Modal
- exactly `1x NVIDIA L4`
- 4 CPU cores
- 16 GiB RAM
- GPU function timeout <= `43,200 s`
- retries `0`
- max billed-compute envelope `$25.00`
- workflow must re-read live rates before allocation and fail closed if the 12-hour worst-case bound exceeds `$25.00`

## Required runner/workflow guards

Implementation must add a dedicated additive runner + workflow and tests that enforce:
- exact source/tree/model/evaluator/prep/runner/workflow blob bindings;
- exact trigger title/body and `run_attempt == 1`;
- unique immutable result root `/vol/chm-v1/100m-stage-c/issue-988/seed-977001-v1`;
- zero-GPU preflight before dispatch;
- frozen corpus fingerprint before allocation and re-check inside GPU function;
- durable pre-allocation dispatch marker;
- entering the scientific GPU function atomically records seed/attempt consumption before training;
- any post-allocation exception is infrastructure failure and the seed remains consumed; never retry/resume/redispatch;
- intermediate checkpoints are audit artifacts only;
- durable authoritative `RESULT.json` or `ATTEMPT_FAILURE.json` is written regardless of Actions status;
- final classification only from #986 pure gate on final step-2048 models;
- no Stage-D automatic execution.

## Authority sequence

1. Implement runner/workflow/tests on a fresh branch.
2. Exact PR-head CI must PASS.
3. Merge through PR with expected-head pin; never write `main` directly.
4. Exact merged-main CI must PASS.
5. Re-audit seed `977001`, trigger uniqueness, result namespace, current main, runner/workflow blobs, and live rate.
6. Post one final launcher-authority comment on this issue binding all exact identities and explicitly authorizing exactly one trigger.
7. Only after that comment may the unique trigger issue be created once.
8. No Actions rerun, retry, rescue run, redispatch, or checkpoint resume after allocation begins.

## Interpretation ceiling

A successful execution is still a one-seed ~100M scaling screen. The only scientific classifications are the already frozen:
- `CHM_V1_100M_STAGE_C_POSITIVE_SCALING_SCREEN`, or
- `CHM_V1_100M_STAGE_C_STOP_SCALE_BRANCH`.

Infrastructure failure is not scientific evidence. A positive screen may justify preparing, not automatically executing, a fresh confirmatory protocol. No novelty/SOTA/AGI/RSI/breakthrough claim is authorized from this screen alone.

**Comment by vinceackermann2-sys** [comment]
CHM_V1_990_FINAL_LAUNCHER_AUTHORITY_V1

source_sha=c430964a9bd3f678b1820ed3dbf6b5c9766ec262
source_tree=e0ead84f5575b135b6b5bd5a0ddd7455d6d051c5
model_blob=b9b141c0e52d4fd0fff28b12a3588b2adc659b8f
evaluator_blob=863bd038e60da5511503adb0c8e1046a680ed3bd
prep_blob=bc4ed60885aaf991a2d6b9fe8f634ff973f3f722
core_blob=26668b37a6062c641275e177b622948b36d0f227
runner_blob=0889018069bb9b0cb27335ac1a108593aabcc51e
implementation_blob=fb7fe5f8c2a597f7de9e4d03c6908dc0db9558ac
packaged_implementation_blob=fb7fe5f8c2a597f7de9e4d03c6908dc0db9558ac
workflow_blob=2fae3fa05a8e0fd19c4f27f33a8724bb6b9d5b1e
authority_audit_workflow_blob=c1f5eac4d59c65a630128239a364e6d5133464db
ci_status=PASS
exact_main_ci_run=35139815314
exact_main_ci_tests=1468
authority_audit_v3_run=35265492305
authority_audit_v3_status=PASS
authority_audit_comment_id=5720084334
scientific_seed=977001
gpu=1x NVIDIA L4
max_gpu_seconds=43200
max_stage_c_compute_usd=25.00
live_hourly_resource_usd=1.117200000000
live_12h_worst_case_usd=13.406400000000
result_namespace_unused=true
trigger_unique_precreation=true
automatic_retry_authorized=false
checkpoint_resume_authorized=false
trigger_authorized=true
stage_d_automatic_authority=false

Authority: exactly one post-authority trigger issue titled `[modal-chm-v1-100m-stage-c-988-seed-977001-v1]` may be created for this frozen Stage-C one-seed scientific screen. No Actions rerun, retry, rescue, redispatch, or checkpoint resume is authorized after allocation begins. Infrastructure failure remains non-scientific evidence and consumes seed 977001 once the scientific GPU function begins.
